modules/nf-core/crabz/compress/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+nextflow_process {
+
+    name "Test Process CRABZ_COMPRESS"
+    script "../main.nf"
+    process "CRABZ_COMPRESS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "crabz"
+    tag "crabz/compress"
+
+    test("sarscov2 - genome - fasta") {
+
+
+        when {
+            process {
+                """
+
+                input[0] = [
+                    [ id:'test'], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
I am unsure why your md5 sums change but it might be possible to check for something else. For example if the first lines of the file stay the same?
Something along
`{ assert snapshot(file(process.out.stats.get(0).get(1)).readLines()[0..5]).match() }`
`{ assert path(process.out.candidate_small_indels_vcf.get(0).get(1)).linesGzip.contains("##fileformat=VCFv4.1") }`
(you might need to adapt the syntax though).